src/canonicalBookMap/sociology.ts

```diff
@@ -7,197 +7,393 @@ export default {
   /* Introduction to Sociology 1e */ 'afe4332a-c97f-4fc4-be27-4e4d384a32d8': [
     /* Introduction to Sociology 2e */ ['02040312-72c8-441e-a685-20e9333f3e1d', {
```

Comment:
This should be `3e` and id of `3e` version

Reply:
We should probably also fix comments to contain only names of the old modules

Reply:
Do we want to change the comments for entire sociology file or the other as well?

Reply:
I would change comments only for this file